src/edge_kernels/MomentumEdgeSolverAlg.C

```diff
@@ -196,9 +234,9 @@ MomentumEdgeSolverAlg::execute()
         const DblType uiIp = 0.5 * (vel.get(nodeR, i) + vel.get(nodeL, i));
 
         // Upwind contribution
-        const DblType uiUpw = (mdot > 0.0)
-                                ? (alphaUpw * uIpL[i] + om_alphaUpw * uiIp)
-                                : (alphaUpw * uIpR[i] + om_alphaUpw * uiIp);
+        const DblType uiUpw =
+          (mdot > 0.0) ? (alphaUpw_w_vof * uIpL[i] + om_alphaUpw_w_vof * uiIp)
```

Comment:
the suffix `_vof` is confusing to me. Can we keep it as it is and just modify the value if vof is on? My thinking is that if I read this code I am thinking that it is vof specific.

Reply:
Yeah I agree. It seems like the vof changes are to modifying the general upwinding?  If there is not a second set of behavior that requires these variables for vof then I think it makes sense to just remove these variables and modify the original as needed when VOF is active.

Reply:
It ends up being identical to alphaUpw and om_alphaUpw if a non-VOF case is run, or you are not at the interface with VOF. I'll change over the naming.